shared/message/vision/Ball.proto

```diff
@@ -23,30 +23,23 @@ package message.vision;
 
 import "google/protobuf/timestamp.proto";
 import "Vector.proto";
-import "message/Geometry.proto";
 import "Matrix.proto";
 
 message Ball {
-    enum MeasurementType {
-        UNKNOWN      = 0;
-        REPROJECTION = 1;
-        WIDTH_BASED  = 2;
-    }
-    message Measurement {
-        MeasurementType type = 1;
-        /// Vector from camera to ball in camera coordinates
-        fvec3 srBCc      = 2;  // Spherical Reciprocal Coordinates (1/distance, phi, theta)
-        fmat3 covariance = 3;
-    }
-    /// Cone describing the ball from the camera's perspective
-    Cone                 cone         = 1;
-    repeated Measurement measurements = 2;
+    /// Central axis unit vector of the ball from the camera in camera space
+    fvec3 uBCc = 1;
+    // Spherical Reciprocal Coordinates (1/distance, phi, theta)
+    fvec3 srBCc = 2;
```

Comment:
Is there a particular reason we want to store it in spherical reciprocal coordinates (and from the camera) rather than something like `rBCw` (or `rBCc`)?

Reply:
The distance measurement scales better in reciprocal space, cant remember why we want spherical. @TrentHouliston ?

Reply:
The reason is that it's the domain of the covariance.
So when our measurements have a covariance value that corresponds to the spherical coordinates. Because our error is in the angular position, and reciprocal distance gives an ok approximation of making the error curve in the distance look like a normal distribution. (normally the further you go the greater your error is which isn't very normal)